SPECS/xcp-ng-generic-lib.spec

```diff
@@ -0,0 +1,57 @@
+Name:           xcp-ng-generic-lib
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        A library of algorithms, I/O and networking functions...
```

Comment:
No terminating punctuation in Summary usually

Reply:
No problem.